### PR TITLE
Document charm libs and provide command to update

### PR DESCRIPTION
**Pull request by samuelallan72**

### CODE REVIEW

**Review by chanchiwai-ray — APPROVED** [review]
LGTM

File: tox.ini [hunk 1]
```diff
@@ -59,3 +59,9 @@ deps =
     mypy
 commands =
     mypy --install-types --non-interactive .
+
+[testenv:update-charm-lib]
```
Comment (jneo8) [inline tox.ini:63]: **issue (blocking):** 

The command indirect result is the creation of an additional Python environment within .tox solely for its execution. If storage for the command is needed, a Makefile would be a more appropriate choice

**Review by jneo8 — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (chanchiwai-ray) [reply tox.ini:63]: A script like previously will be fine (e.g. `scripts/update-charm-lib.sh`), using makefile for just a single target is overkill.

Comment (chanchiwai-ray) [reply tox.ini:63]: unless you have more targets in mind?

Comment (chanchiwai-ray) [reply tox.ini:63]: we won't need scripts for update dashboard files from openstack-exporter-k8s, it's very simple commands, can be in the workflow itself

Comment (chanchiwai-ray) [reply tox.ini:63]: that being said, I don't oppose plan for the future

Comment (chanchiwai-ray) [reply tox.ini:63]: Sure

Comment (chanchiwai-ray) [reply tox.ini:63]: well, we can make it independent though, we create `scripts/*.sh` and let makefile to call the scripts

Comment (jneo8) [reply tox.ini:63]: > well, we can make it independent though, we create scripts/*.sh and let makefile to call the scripts

This one or makefile are work for me.

**Review by chanchiwai-ray — APPROVED** [review]

**Review by jneo8 — APPROVED** [review]

**Comment by chanchiwai-ray** [comment]
Rebasing with web UI create a `Unverified` commit and blocks the merge ¯\_(ツ)_/¯

**Pull request merged**